pkg/omf/functions/search/omf.search.fish

```diff
@@ -0,0 +1,49 @@
+function omf.search -d "Search db for a certain package or theme"
+
+  switch (count $argv);
+    case 1;
+      __omf.search.pkg $argv[1]
+      echo
+      __omf.search.theme $argv[1]
+    case 2;
+      switch "$argv[1]"
+        case "-pkg" "--package";
+          __omf.search.pkg $argv[2]
+        case "-t" "--theme";
+          __omf.search.theme $argv[2]
+        case '*';
+          __omf.search.usage
+      end
+    case '*';
+      __omf.search.usage
```

Comment:
The idea is that all command line interface stuff like parameter should be done in `omf.cli.*` functions. Please move this "0 arguments" case there :+1: